### PR TITLE
🌐 [i18n-KO] Translated`tasks/document_question_answering.md` to Korean

### PR DESCRIPTION
**Pull request by jungnerd**
# What does this PR do?

Translated the `tasks/document_question_answering.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? 
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@sgugger, @ArthurZucker, @eunseojo May you please review this PR?

### CODE REVIEW

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
It was easy to understand, thanks to the careful translation of the annotations👍
Thanks for the good translation!
I commented below a review related to glossary.

File: docs/source/ko/tasks/document_question_answering.md [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+
+```py
+>>> model_checkpoint = "microsoft/layoutlmv2-base-uncased"
+>>> batch_size = 4
+```
+
+## 데이터 불러오기 [[load-the-data]]
+
+이 가이드에서는 🤗 Hub에서 찾을 수 있는 전처리된 DocVQA의 작은 샘플을 사용합니다. If you'd like to use the full
+DocVQA의 전체 데이터 세트를 사용하고 싶다면, [DocVQA homepage](https://rrc.cvc.uab.es/?ch=17)에 가입 후 다운로드 할 수 있습니다. 전체 데이터 세트를 다운로드 했다면, 이 가이드를 계속 진행하기 위해 [🤗 dataset에 파일을 로드하는 방법](https://huggingface.co/docs/datasets/loading#local-and-remote-files)을 확인하세요.
```
Comment (sim-so) [inline docs/source/ko/tasks/document_question_answering.md:82]:
```suggestion
DocVQA의 전체 데이터 세트를 사용하고 싶다면, [DocVQA homepage](https://rrc.cvc.uab.es/?ch=17)에 가입 후 다운로드 할 수 있습니다. 전체 데이터 세트를 다운로드 했다면, 이 가이드를 계속 진행하기 위해 [🤗 dataset에 파일을 가져오는 방법](https://huggingface.co/docs/datasets/loading#local-and-remote-files)을 확인하세요.
```

File: docs/source/ko/tasks/document_question_answering.md [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+그리고 주석 작성자가 제공한 데이터 세트에서 첫 번째 답변을 가져옵니다. 또는 무작위로 샘플을 추출할 수도 있습니다.
+
+```py
+>>> updated_dataset = dataset.map(lambda example: {"question": example["query"]["en"]}, remove_columns=["query"])
+>>> updated_dataset = updated_dataset.map(
+...     lambda example: {"answer": example["answers"][0]}, remove_columns=["answer", "answers"]
+... )
+```
+
+이 가이드에서 사용하는 LayoutLMv2 체크포인트는 `max_position_embeddings = 512`로 훈련되었습니다(이 정보는 [checkpoint's `config.json` file](https://huggingface.co/microsoft/layoutlmv2-base-uncased/blob/main/config.json#L18)에서 확인할 수 있습니다).
```
Comment (sim-so) [inline docs/source/ko/tasks/document_question_answering.md:125]:
```suggestion
이 가이드에서 사용하는 LayoutLMv2 체크포인트는 `max_position_embeddings = 512`로 훈련되었습니다(이 정보는 [체크포인트의 `config.json` 파일](https://huggingface.co/microsoft/layoutlmv2-base-uncased/blob/main/config.json#L18)에서 확인할 수 있습니다).
```

File: docs/source/ko/tasks/document_question_answering.md [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+<!--This tip is automatically generated by `make fix-copies`, do not fill manually!-->
+
+[LayoutLM](../model_doc/layoutlm), [LayoutLMv2](../model_doc/layoutlmv2), [LayoutLMv3](../model_doc/layoutlmv3)
+
+<!--End of the generated tip-->
+
+</Tip>
+
+LayoutLMv2는 토큰의 마지막 은닉층 위에 질의 응답 헤드를 추가해 답변의 시작 토큰과 끝 토큰의 위치를 예측함으로써 문서 질의 응답 태스크를 해결합니다. 즉, 문맥이 주어졌을 때 질문에 답하는 정보를 추출하는 추출형 질의 응답(Extractive question answering)으로 문제를 처리합니다.
+문맥은 OCR 엔진의 출력에서 가져오며, 여기서는 Google의 테서랙트(Tesseract)를 사용합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/document_question_answering.md:43]:
```suggestion
문맥은 OCR 엔진의 출력에서 가져오며, 여기서는 Google의 Tesseract를 사용합니다.
```
Transformers를 한국어로 쓰지 않는 규칙을 Tesseract에도 적용해야 하지 않을까 생각해 보았습니다. 
참고 부탁 드립니다!

File: docs/source/ko/tasks/document_question_answering.md [hunk 4]
```diff
@@ -0,0 +1,482 @@
...
+
+이미지에 OCR을 적용했으면 데이터 세트의 텍스트 부분을 모델에 맞게 인코딩해야 합니다.
+이 인코딩에는 이전 단계에서 가져온 단어와 박스를 토큰 수준의 `input_ids`, `attention_mask`, `token_type_ids` 및 `bbox`로 변환하는 작업이 포함됩니다. 
+텍스트를 전처리하려면 프로세서의 `tokenizer`가 필요합니다.
+
+```py
+>>> tokenizer = processor.tokenizer
+```
+
+위에서 언급한 전처리 외에도 모델울 위해 레이블을 추가해야 합니다. 🤗 Transformers의 `xxxForQuestionAnswering` 모델의 경우, 레이블은 `start_positions`와 `end_positions`로 구성되며 어떤 토큰이 답변의 시작과 끝에 있는지를 나타냅니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/document_question_answering.md:202]:
```suggestion
위에서 언급한 전처리 외에도 모델 위해 레이블을 추가해야 합니다. 🤗 Transformers의 `xxxForQuestionAnswering` 모델의 경우, 레이블은 `start_positions`와 `end_positions`로 구성되며 어떤 토큰이 답변의 시작과 끝에 있는지를 나타냅니다.
```

**Review by sim-so — CHANGES_REQUESTED** [review]
몇 가지 수정 의견 보내 드리오니 참고 부탁 드립니다.
고생 많으셨습니다! 😸

File: docs/source/ko/tasks/document_question_answering.md (hunk 4 above)
Comment (wonhyeongseo) [reply docs/source/ko/tasks/document_question_answering.md:202]: 울 -> 을

Comment (wonhyeongseo) [reply docs/source/ko/tasks/document_question_answering.md:202]: 울 -> 을

**Review by jungnerd — COMMENTED** [review]
fix: resolve suggestions

**Review by jungnerd — COMMENTED** [review]
fix: resolve suggestions

**Comment by jungnerd** [comment]
May you please review this PR? 🤗
@sgugger, @ArthurZucker, @eunseojo

**Review by sgugger — APPROVED** [review]
Thanks!

**Pull request merged**